optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -267,12 +267,15 @@ def render(self, result, job):
 
         open_browser = getattr(job.args, 'open_browser', False)
         if getattr(job.args, 'html_job_result', 'off') == 'on':
-            html_dir = os.path.join(job.logdir, 'html')
-            if os.path.exists(html_dir):    # update the html result if exists
-                shutil.rmtree(html_dir)
-            os.makedirs(html_dir)
-            html_path = os.path.join(html_dir, 'results.html')
+            # FIXME: remove html legacy dir after 52.0 LTS release
```

Comment:
Excellent, after the release symlink goes away and this gets queued into the release notes for the future LTS